README.md

```diff
@@ -96,8 +96,6 @@ object MainApp extends App {
 
   val pineconeOptions = Map(
     PineconeOptions.PINECONE_API_KEY_CONF -> apiKey,
-    PineconeOptions.PINECONE_ENVIRONMENT_CONF -> environment,
```

Comment:
Remove environment and projectName variables as input parameters since they are not required to build the index endpoint. Instead, we get the index host using describe index call.